src/sync/gll.ts

```diff
@@ -1,10 +1,30 @@
+import { EventEmitter } from 'events';
 import PQueue from 'queue-promise';
 import { ILogger } from '../types';
 import { GlobalLoadLockTaskError } from '../errors';
 
-const MAX_CONCURRENT_LOAD_TASKS = 5;
+const MAX_CONCURRENT_LOAD_TASKS = 3;
```

Comment:
This increases the throughput of each wallet since each competing wallet splits IO evenly.
I calibrated to 5 when using the wallet-lib but using the headless the highest thorughput comes when using 3.